src/plots/Label/CMakeLists.txt

```diff
@@ -88,6 +87,7 @@ ${VISIT_INCLUDE_DIR}/visit_vtk/full
 ${VISIT_INCLUDE_DIR}/visit_vtk/lightweight
 ${VTK_INCLUDE_DIRS}
 ${PYINCLUDES}
+${VISIT_INCLUDE_DIR}/avt/Plotter/vtk
```

Comment:
Just out of curiosity, why does the order matter here?

Reply:
The change in order is a result of running xml2cmake. Because this is a special include added by CXXFLAGS in the xml file, it gets added to the end of the standard includes.